rma_reason/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,2 @@
+- Jacques-Etienne Baudoux - BCIM <je@bcim.be>
+- Souheil Bejaoui - ACSONE SA/NV <souheil.bejaoui@acsone.eu.com>
```

Comment:
```suggestion
- Souheil Bejaoui - ACSONE SA/NV <souheil.bejaoui@acsone.eu>
```

@sbejaoui

Reply:
For reference, I avoid to put the emails on the CONTRIBUTORS list for avoiding direct requests and spam. The email is already in the commit.